Bio/Entrez/Parser.py

```diff
@@ -328,7 +328,7 @@ def read(self, handle):
         if hasattr(handle, "closed") and handle.closed:
             # Should avoid a possible Segmentation Fault, see:
             # http://bugs.python.org/issue4877
-            raise IOError("Can't parse a closed handle")
+            raise ValueError("Can't parse a closed handle")
```

Comment:
Was there a past discussion about the best exception here?

Reply:
https://github.com/biopython/biopython/pull/2455#discussion_r361974429

Reply:
+1, Bow's argument that we should follow Python 3's example here makes sense to me.